### PR TITLE
virtctl create clone: minor improvements

### PR DESCRIPTION
**Pull request by iholder101**
**What this PR does / why we need it**:
virtctl create clone command was introduced not long ago in this PR: https://github.com/kubevirt/kubevirt/pull/9596.

After the PR had merged, I received some further feedback. This follow-up addresses this feedback.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virtctl/create/clone/clone.go [hunk 1]
```diff
@@ -94,14 +94,15 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringArrayVar(&c.newMacAddresses, NewMacAddressesFlag, nil, "Specify clone's new mac addresses. For example: 'interfaceName0:newAddress0'")
 	cmd.Flags().StringVar(&c.newSmbiosSerial, NewSMBiosSerialFlag, emptyValue, "Specify the clone's new smbios serial")
 
-	_ = cmd.MarkFlagRequired(SourceNameFlag)
+	err := cmd.MarkFlagRequired(SourceNameFlag)
```
Comment (lyarwood) [inline pkg/virtctl/create/clone/clone.go:97]: nit - inline the error check

**Review by lyarwood — COMMENTED** [review]
/lgtm

A silly nit about `err` checking not being inline but otherwise looks great thanks

File: pkg/virtctl/create/instancetype/instancetype.go [hunk 2]
```diff
@@ -96,8 +96,16 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringArrayVar(&c.gpus, GPUFlag, c.gpus, "Specify the list of vGPUs to passthrough. Can be provided multiple times.")
 	cmd.Flags().StringArrayVar(&c.hostDevices, HostDeviceFlag, c.hostDevices, "Specify list of HostDevices to passthrough. Can be provided multiple times.")
 
-	_ = cmd.MarkFlagRequired(CPUFlag)
-	_ = cmd.MarkFlagRequired(MemoryFlag)
+	err := cmd.MarkFlagRequired(CPUFlag)
```
Comment (alicefr) [inline pkg/virtctl/create/instancetype/instancetype.go:99]: nit: inline error checking as suggested above

**Comment by alicefr** [comment]
/approve
/hold

Hold to address the comments

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks!

**Comment by iholder101** [comment]
Thanks for reviewing!
Addressed all comments :)

/hold cancel
@0xFelix @lyarwood

**Pull request merged**